webapp/schemas.py

```diff
@@ -31,3 +32,12 @@ class RemoveWebpageModel(BaseModel):
     due_date: str = ""
     reporter_id: int = None
     description: str = ""
+
+    @validator("due_date")
+    def date_validation(cls, value: str) -> str:
+        try:
+            if datetime.strptime(value, "%Y-%m-%d") <= datetime.now():
+                raise ValueError("due date must be in the future")
+            return value
+        except ValueError:
```

Comment:
A few issues here:
- We don't need to catch the `ValueError` at all here, as pydantic handles raised `ValueError` and `AssertionError` by default
- we should use the newer `field_validator` or `model_validator` since we have an updated version of Pydantic

Example [here](https://docs.pydantic.dev/latest/concepts/validators/#handling-errors-in-validators)